src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -0,0 +1,133 @@
+//package seedu.address.storage;
```

Comment:
this file will be morphed into JsonAdaptedResidenceTest?

Reply:
Yup!